### ISSUE TITLE
This plugin will stop working soon

### ISSUE THREAD

**Issue by olsh** [olsh/Feedly-Notifier#205]
It'll probably happen on the ~6th of April 2022.

The reason why this happens is not technical or lack of interest. The reason is that _I'm Russian_.
I cannot renew the domain (olsh.me) that is used for the authentication and image server, MasterCard & Visa cards issued by _Russian banks_ are blocked everywhere, and PayPal is blocked _for all Russians_. So I have no options to renew the domain. I've tried to contact my USA-based registrar with no luck.
I'm not going to work around this because rules are rules. And if this _discrimination by nation_ will **stop this war**, let it be.

Bottom line, my suggestion is to find an alternative or fork the extension if you need it. 
☮️
